src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,572 @@
+             'to be imported into Mondo.')
+    parser.add_argument(
+        '-O', '--outpath-added-obsolete', required=False, default=EX_DEFAULTS['outpath_added_obsolete'],
+        help='Creates robot template containing new subclass relationships from given ontology that would be imported '
+             'into Mondo, except for that these terms are obsolete in Mondo.')
+    parser.add_argument(
+        '-c', '--outpath-confirmed', required=False, default=EX_DEFAULTS['outpath_confirmed'],
+        help='Path to output robot template containing subclass relations for given ontology that exist in Mondo and '
+             'are confirmed to also exist in the source.')
+    parser.add_argument(
+        '-M', '--outpath-direct-in-mondo-only', required=False, default=EX_DEFAULTS['outpath_in_mondo_only'],
+        help='Path to create file for relations for given ontology where direct subclass relation exists only in Mondo'
+             ' and not in the source.')
+    parser.add_argument(
+        '-o', '--onto-config-path', required=False, default=EX_DEFAULTS['onto_config_path'],
+        help='Path to a config `.yml` for the ontology which contains a `base_prefix_map` which contains a '
+             'list of prefixes owned by the ontology. Used to filter out terms.')
+    parser.add_argument(
+        '-d', '--mondo-ingest-db-path', required=False, default=EX_DEFAULTS['mondo_ingest_db_path'],
+        help='Path to SemanticSQL sqlite  `mondo-ingest.db`.')
+    parser.add_argument(
+        '-D', '--mondo-db-path', required=False, default=EX_DEFAULTS['mondo_db_path'],
+        help='Path to SemanticSQL sqlite `mondo.db`.')
+    parser.add_argument(
+        '-m', '--mondo-mappings-path', required=False, default=EX_DEFAULTS['mondo_mappings_path'],
+        help='Path to file containing all known Mondo mappings, in SSSOM format.')
+    # Alternative CLI for combining --outpath-direct-in-mondo-only
+    parser.add_argument(
+        '-C', '--collate-direct-in-mondo-only', required=False,
+        help='Combine all --outpath-direct-in-mondo-only outputs for all sources at given path, using those as inputs, and '
+             'then deletes them after. This flag should only be used by itself.')
+    parser.add_argument(
+        '-u', '--use-cache', required=False, default=EX_DEFAULTS['use_cache'],
+        help='Use cached outputs, if any present, for outputs from operations that fetch ancestors. Cache will always'
+             ' be saved even if this flag is not present, but it will not be loaded from unless flag is present.')
+
+    d = vars(parser.parse_args())
+    if d['collate_direct_in_mondo_only']:  # todo: Maybe make the collation part a separate script?
+        return collate_direct_in_mondo_only(d['collate_direct_in_mondo_only'])
+    del d['collate_direct_in_mondo_only']
+    sync_subclassof(**d)
+
+
+def run_defaults(use_cache=True):  # todo: #remove-temp-defaults
```

Comment:
## Default settings @matentzn 
Review and decide.

<details><summary>Details</summary>
<p>

Basically there are some toggleable default settings that can be run when running the Python script directly rather than in the makefile.

This is very helpful during development. It helps me quickly run and debug things. And there are some things I can do with this defaults setup that I can't do using a debug config with saved params.

We've discussed this type of thing in the past and you've indicated that you prefer to leave out such defaults code from the final product. That's what this `todo` is. It's an indicator that this stuff should be removed when development is done.

Of course, this feature is too big for a single PR, and there are perhaps it will undergo development for many months, so I'd like to leave the defaults there so that I can toggle their usage when I develop further.

Note that these are off by default. So they will have no effect unless I'm doing local development.

Options:
a. Delete them now; Will slow Joe down when he continues to develop in the future, but he can temporarily add them back in another PR.
b. Leave them for now. Delete at some point in the future when other cases are completed.
c. Simply leave them indefinitely into the future. They are no harm in production, and useful for further development and debugging.

</p>
</details>

Reply:
Thank you for the detailed analysis. This is TMD for me, I will trust you to make the right call!